### DISCUSSION TITLE
Passaggio da cavo a rete wifi HA su Mini Pc

### DISCUSSION THREAD

**Discussion by giuseppe-mancuso** [indomus/forum#3204] [HUB: Home Assistant]
Ciao a tutti,

ho cercato un po' in giro ma probabilmente con le keyword sbagliate: ho installato HA su un mini PC collegando il cavo di rete. Funziona tutto alla grande. A questo punto vorrei passare alla rete wifi: ho provato con la chiave USB e il riavvio del PC ma sembra che vado tutto bene ma poi HA rest offline. C'è qualche modo di farlo tramite interfaccia?

Grazie

**Comment by Scialla** [comment]
A parte che non si capisce cosa c'entri la chiave USB con il WiFi (che presumo sia integrato nel miniPC), è possibile che, se hai installato Debian, la scheda WiFi non sia stata riconosciuta e non siano stati installati i driver. Verifica la cosa e, se così fosse, devi sbatterti a trovare soluzioni online per vedere come installare i driver e abilitarla.

**Reply by KC-inDomus** [reply]
sta sicuramente usando home assistant os e fa riferimento alla chiavetta usb che ti prepari al primo boot per passargli le coordinate wi-fi.
è più facile di così, @giuseppe-mancuso: vai nelle impostazioni di rete di home assistant e imposta il ssdi e password wifi, tutto qui.
